tutorials/Explanation.ipynb

```diff
@@ -43,7 +43,8 @@
     "- [MAE](./Mean_Absolute_Error.ipynb) / [RMSE](./Root_Mean_Squared_Error.ipynb) / [MSE](./Mean_Squared_Error.ipynb)\n",
     "- [Murphy Diagrams](./Murphy_Diagrams.ipynb)\n",
     "- [Pearson's Correlation](./Pearsons_Correlation.ipynb)\n",
-    "- [Quantile Loss](./Quantile_Loss.ipynb)\n"
+    "- [Quantile Loss](./Quantile_Loss.ipynb)\n",
+    "- [Bias](./Bias.ipynb)"
```

Comment:
In the Bias tutorial, it says "Additive bias is often called the "mean error" and can tell us the average error. It does not tell us the magnitude of the error." . I'm a bit unclear what that's supposed to convey. Multiplicative bias doesn't give a magnitude as it's a ratio, but additive bias does have a magnitude. I would also give some brief examples. I've been wracking my brain trying to think of simple real-world examples of each and coming up short.

Reply:
Further in the tutorial, where the temperature clipping occurs, I would suggest adding something to the effect "to avoid the effort of downloading further sample data, we will modify the temperature data to stand in for wind speed. Readers may wish to explore the data available from the NCI server and download a wind grid instead for this example".

Reply:
Also, presumably there are other distribution types, like a power law bias that could even occur? Are they useful to consider?

Reply:
You can have a bias of zero, but still have an error with a non-zero magnitude. MAE would measure this. I've updated the text.

Added comment about not downloading more data.

Yes, there are other forms of bias (e.g., the bias correction to minimise the mean Huber Loss). I think other forms of bias can be implemented on an as needed basis